alpine/2.3.6/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Only alpine version that suports rethinkdb is edge
+FROM alpine:edge
+
```

Comment:
Needs a maintainer line. Right now, all Dockerfiles list @dalanmiller as maintainer (example: https://github.com/rethinkdb/rethinkdb-dockerfiles/blob/master/jessie/2.3.6/Dockerfile#L3), but, per https://github.com/docker-library/official-images/pull/3332#issuecomment-322614740, we should probably change these to @AtnNn. (For now, I'd stick with copying the @dalanmiller credit, to make one-shot updating everything simpler.)